### PR TITLE
(http): update docs and add response headers to reject path

### PR DESCRIPTION
**Pull request by henri-hulski**

### CODE REVIEW

**Comment by henri-hulski** [comment]
I added response headers also to reject path.
This makes sense e.g. for status code 401 which requires a WWW-Authenticate header field.

> **401 Unauthorized** Similar to 403 Forbidden, but specifically for use when authentication is required and has failed or has not yet been provided. The response must include a WWW-Authenticate header field containing a challenge applicable to the requested resource.

File: packages/cerebral-provider-http/README.md [hunk 1]
```diff
@@ -170,12 +170,15 @@ function someGetAction ({http}) {
     .then((response) => {
       response.status // Status code of response
       response.result // Parsed response text
+      // The response headers are returned as an object with lowercase header
+      // names as keys. Values belonging to the same key are seperatet by ', '.
```
Comment (Guria) [inline packages/cerebral-provider-http/README.md]: seperatet -> separated

Comment (henri-hulski) [inline packages/cerebral-provider-http/README.md]: @Guria Fixed.

**Comment by Guria** [comment]
Thanks :)

**Pull request merged**